valohai_yaml/utils/parse.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Dict
+
+from valohai_yaml.objs.environment_variable import EnvironmentVariable
+from valohai_yaml.objs.input import Input
+from valohai_yaml.objs.mount import Mount
+from valohai_yaml.objs.parameter import Parameter
+from valohai_yaml.objs.utils import consume_array_of
+from valohai_yaml.types import SerializedDict
+
+
+def parse_common_kwargs(data: SerializedDict) -> Dict[str, Any]:
```

Comment:
Mmm, these are common to... steps and overrides, I guess? Could we find a better name and place for this?

Reply:
parse_step_properties(or _fields)? It is 'step' because override works with it. What do you think?

Reply:
`parse_common_step_properties` sounds pretty good. Please also add a docstring then! :)